src/httpserver/handlers/handler_function_selector.clj

```diff
@@ -0,0 +1,8 @@
+(ns httpserver.handlers.handler_function_selector
+  (:require [httpserver.handlers.ok :as ok]
+            [httpserver.handlers.teapot :as teapot]))
+
+(defn get-function [request]
+  (cond
+    (= true (= (= "GET" (:method request)) (= "/coffee" (:path request)))) teapot/call
```

Comment:
Do you have an extra call to `=` here? Hard to tell. I think some private functions would help readability here